conf/public_aws_ecr.config

```diff
@@ -8,8 +8,9 @@
 ----------------------------------------------------------------------------------------
 */
 
-docker.registry = 'public.ecr.aws'
-podman.registry = 'public.ecr.aws'
+docker.registry      = 'public.ecr.aws'
+podman.registry      = 'public.ecr.aws'
+singularity.registry = 'public.ecr.aws'
```

Comment:
Will singularity ever be used on AWS in this context? To be fair, will `podman` ever be used either? Maybe having `docker.registry` is enough here?

Reply:
We had people trying to run singularity on AWS, so I'd rather be safe than sorry

Reply:
cf https://nfcore.slack.com/archives/CE6SDBX2A/p1686824156626549